src/GafferUIModule/AnimationGadgetBinding.cpp

```diff
@@ -71,16 +67,21 @@ Gaffer::StandardSetPtr editablePlugs( AnimationGadget &a )
 	return a.editablePlugs();
 }
 
+Gaffer::SetPtr selectedKeys( AnimationGadget &a )
+{
+	return a.selectedKeys();
+}
```

Comment:
I think if we used `return_value_policy<IECorePython::CastToIntrusivePtr>()` we wouldn't need this wrapper function, or the ones above.